app/views/shared/comments/_comments.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="comments-container" data-object-id = <%= object.id %>> 
   <% per_page = Constants::COMMENTS_SEARCH_LIMIT %>
-  <div class="content-comments">
+  <div class="content-comments inline_scroll_block">
```

Comment:
where do you define this `inline_scroll_block` class name? btw, can we use `inline-scroll-block` name?